src/dataExplorer/components/SaveAsScript.tsx

```diff
@@ -79,6 +79,7 @@ const SaveAsScript: FC<Props> = ({onClose, type}) => {
   const clear = useCallback(() => {
     cancel()
     setStatus(RemoteDataState.NotStarted)
+    clearSchemaSelection()
```

Comment:
This was missing, based removed on an earlier Save/Load PR.

On clear, we also need to clear out the session for the schema.
Otherwise the schema composition will try to re-create the editor text based on the session.